packages/cdk8s-cli/lib/import/type-generator.ts

```diff
@@ -348,22 +348,26 @@ export class TypeGenerator {
   }
 }
 
+/**
+ * Convert all-caps acronyms (e.g. "VPC", "FooBARZooFIGoo") to pascal case (e.g. "Vpc", "FooBarZooFiGoo").
+ *
+ * @internal exported for tests
+ */
+export function normalizeTypeName(typeName: string) {
```

Comment:
Why did the old implementation only normalize types that start with `I`? This seems like a pretty major behavior change that i'd like to understand the reasoning behind.

Reply:
Originally I only took care of types that started with `IABCFoo` so that jsii won’t confuse with interfaces, but I’ve realized we should convert everything.

Thanks for caring!

Reply:
Yeah that seems like a better approach 👍